modules/javafx.graphics/src/main/java/javafx/scene/AccessibleAttribute.java

```diff
@@ -800,4 +821,27 @@ public enum AccessibleAttribute {
     public Class<?> getReturnType() {
         return returnClass;
     }
+
+    /**
+     * This enum describes the values for TOGGLE_STATE attribute.
+     *
+     * @see TOGGLE_STATE
```

Comment:
Hopefully, this is just an Eclipse limitation. I'll check the actual javadoc (which I was planning to do anyway).

Reply:
It does work in the actual javadocs (so almost certainly an Eclipse issue).

Minor suggestion: use `{@link ...}` in the Description (and then you can remove the `@see`) like this:

```
This enum describes the values for the {@link TOGGLE_STATE} attribute.
```

Reply:
In Eclipse, only this works:
```
{@link AccessibleAttribute#TOGGLE_STATE}
```

Reply:
Then I think Eclipse has a bug. `TOGGLE_STATE` is a field (enum value) in the outer enum class, `AccessibleAttribute`, so can be referenced from the nested enum class without qualifying it with the outer class name. If we wanted to be more clear (and work around the Eclipse limitation), it could be written as:

```
{@link AccessibleAttribute#TOGGLE_STATE TOGGLE_STATE}
```

Reply:
Changed it as : `This enum describes the values for {@link AccessibleAttribute#TOGGLE_STATE TOGGLE_STATE} attribute.`